### PR TITLE
release: v0.6.0

### PR DESCRIPTION
**Pull request by tiborsimko**
Signed-off-by: Tibor Šimko <tibor.simko@cern.ch>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+--------------------------
+
+- Adds new API for Gitlab integration.
+- Adds new API for Kubernetes ingresses.
+- Adds new APIs for management of user secrets.
+- Adds EOS storage Kubernetes configuration.
+- Adds HTCondor and Slurm compute backends.
+- Adds support for streaming file uploads.
+- Allows unpacked CVMFS and CMS open data volumes.
+- Adds Serial workflow steps.
```
Comment (roksys) [inline CHANGES.rst]: Just to clarify, is it partial workflow execution?

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds new API for Gitlab integration.
+- Adds new API for Kubernetes ingresses.
```
Comment (roksys) [inline CHANGES.rst]: For me, it sounds like it's REANA native API. What do you think about ``Adds new Kubernetes client API for ingresses``?

File: CHANGES.rst (hunk 1 above)
Comment (tiborsimko) [reply CHANGES.rst]: It's for step name and compute backend, I'll reformulate.

File: CHANGES.rst (hunk 2 above)
Comment (tiborsimko) [reply CHANGES.rst]: Changed.

File: tests/test_mq.py [hunk 3]
```diff
@@ -1,23 +1,10 @@
...
-# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
-# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with REANA; if not, see <http://www.gnu.org/licenses>.
-#
-# In applying this license, CERN does not waive the privileges and immunities
-# granted to it by virtue of its status as an Intergovernmental Organization or
-# submit itself to any jurisdiction.
+# REANA is free software; you can redistribute it and/or modify it
```
Comment (tiborsimko) [inline tests/test_mq.py]: Noticed we had GNU GPL is some test places (huh!) for a reason or another; amended all and verified that all other repositories are OK.

**Pull request merged**